docs/docs/making-your-site-accessible.md

```diff
@@ -23,15 +25,32 @@ While ultimately it's up to you to develop your site with accessibility in mind,
+That's why every Gatsby site aims to have an accessible navigation experience by default. Thanks to [@reach/router](https://reach.tech/router), a routing library for React, Gatsby handles page announcements for screen readers on page change. We're actively making improvements to this experience, and we [welcome your feedback](/accessibility-statement/).
 
-Since the [second major release](/blog/2018-09-17-gatsby-v2/), your Gatsby sites use `@reach/router` under the hood. The [Gatsby Link Component](/docs/gatsby-link/) wraps [@reach/router's Link component](https://reach.tech/router/api/Link), so you don't need to think about it.
+Since the [second major release](/blog/2018-09-17-gatsby-v2/), your Gatsby sites use `@reach/router` under the hood. While additional accessibility testing is always a good idea, the [Gatsby Link Component](/docs/gatsby-link/) wraps [@reach/router's Link component](https://reach.tech/router/api/Link) to improve accessibility without you having to think about it.
 
 ## How to improve accessibility?
 
-Accessibility by default is a win for everyone. Learn more about web accessibility in general:
-
-- [Free course](https://www.udacity.com/course/web-accessibility--ud891) by Google and Udacity.
-- [WebAIM introduction](https://webaim.org/intro/) to web accessibility.
+Accessibility by default is a win for everyone. Here's a starting point for accessibility testing when making a Gatsby site or theme:
+
+- [Use your keyboard](https://webaim.org/techniques/keyboard/) to tab through the pages. Can you reach and operate every interactive control (links, buttons, form inputs, etc.) and see a focus indicator on the screen?
+- Use a browser extension like [axe](https://www.deque.com/axe/) or [Accessibility Insights](https://accessibilityinsights.io/) to find and fix common accessibility issues in development
+- Test for [adequate color contrast](https://dequeuniversity.com/tips/color-contrast) with the [accessibility color picker in Chrome Developer Tools](https://developers.google.com/web/updates/2018/01/devtools#contrast)
+- Create inclusive and [accessible forms](/docs/building-a-contact-form#creating-an-accessible-form)
+- Employ accessible [headings, landmarks, and semantic structure](https://webaim.org/techniques/semanticstructure/)
+- Include [image, video, and audio text alternatives](https://a11y-style-guide.com/style-guide/section-media.html)
+- Test for [screen magnification and zoom](https://axesslab.com/make-site-accessible-screen-magnifiers/)
+- Ensure accessibility of [interactive menus, modals, and custom widgets](https://developer.mozilla.org/en-US/docs/Web/Accessibility/An_overview_of_accessible_web_applications_and_widgets)
+- Create safe [animations and motion](https://alistapart.com/article/designing-safer-web-animation-for-motion-sensitivity/)
+- Write [Cypress accessibility tests](/docs/end-to-end-testing/#writing-tests) for your site or application
+
+## Accessibility resources
```

Comment:
What about the A11y Project quick tests / quick tips?

https://a11yproject.com/#Quick-tests